src/shogun/util/PutPerceptron_benchmark.cc

```diff
@@ -0,0 +1,137 @@
+#include <benchmark/benchmark.h>
+#include <shogun/features/DenseFeatures.h>
+#include <shogun/features/DotFeatures.h>
+#include <shogun/features/iterators/DotIterator.h>
+#include <shogun/labels/BinaryLabels.h>
+#include <shogun/mathematics/Math.h>
+
+namespace shogun
+{
+	// Mock PerceptronP
+	class PerceptronP : public CSGObject
```

Comment:
PerceptronP? What does the P stand for?

Reply:
is this really meant to be merged into develop? or rather have it in a feature branch?